maths/remove_digit.py

```diff
@@ -0,0 +1,38 @@
+def remove_digit(num: int) -> int:
+    """
+
+    returns the biggest possible result
+    that can be achieved by removing
+    one digit from the given number
+
+    >>> remove_digit(152)
+    52
+    >>> remove_digit(6385)
+    685
+    >>> remove_digit(-11)
+    1
+    >>> remove_digit(2222222)
+    222222
+    >>> remove_digit("2222222")
+    Traceback (most recent call last):
+    TypeError: only integers accepted as input
+    >>> remove_digit("string input")
+    Traceback (most recent call last):
+    TypeError: only integers accepted as input
+    """
+
+    if isinstance(num, int):
```

Comment:
you could switch this around in such a manner that the typeerror is raised at the start if the instance is incorrect

Reply:
hi! thank you so much for your feedback. just implemented it! however since this file for some [ unknown ] reason went to another one of my pull requests, i commited the fixes to the pr #6708 :) appreciate the feedback!!